packages/svelte/src/internal/client/runtime.js

```diff
@@ -371,17 +371,28 @@ export function handle_error(error, effect, previous_effect, component_context)
  * @param {Effect} effect
  * @param {number} [depth]
  */
-function schedule_possible_effect_self_invalidation(signal, effect, depth = 0) {
+function schedule_possible_effect_self_invalidation(
+	signal,
+	effect,
+	depth = 0,
```

Comment:
Can we use a boolean instead of a number here? Seems like it's only used within one if block